common/namespace/handler_test.go

```diff
@@ -70,7 +71,7 @@ type (
 	}
 )
 
-var nowInt64 = time.Now().UnixNano()
+var now = time.Date(2020, 8, 22, 1, 2, 3, 4, time.UTC)
```

Comment:
Any reason to change from time.Now to a specific date?

Reply:
Yes, I left another comment on it: https://github.com/temporalio/temporal/pull/632#discussion_r462763645